### PR TITLE
COMMERCE-10066

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-10066](https://liferay.atlassian.net/browse/COMMERCE-10066)

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -1209,6 +1209,51 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViweOnlyCorrectAllowedQuantitiesAreSelectableInProductCard {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:1214]: CanViewOnlyCorrectAllowedQuantitiesAreSelectableForProduct

Please combine these 2 tests and make an assertion first on the product card, then on the product details page. It's a repetition and can be worked under one test. Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase [hunk 2]
```diff
@@ -2439,6 +2439,55 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViweOnlyCorrectAllowedQuantitiesAreSelectableInProductDetails {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase:2444]: CanViewOnlyCorrectAllowedQuantitiesAreSelectableForProduct

Please combine these 2 tests and make an assertion first on the product card, then on the product details page. It's a repetition and can be worked under one test. Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 3]
```diff
@@ -1209,6 +1209,51 @@ definition {
...
+
+			for (var number : list "2,6") {
+				AssertElementPresent(
+					key_optionValue = ${number},
+					key_productName = "U-Joint",
+					locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD_OPTION_VALUE");
+			}
+		}
+
+		task ("Then only the quantities that meet the configuration are displayed") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:1247]: Then only the quantities that meet the configuration are displayed on the product card

**Review by baharturk — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:stop:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 4]
```diff
@@ -1209,6 +1209,103 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViewOnlyCorrectAllowedQuantitiesAreSelectableForProduct {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:1214]: Please specify in the description and test name that the test checks the quantities in the product card

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:forward

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
Failures are not related to the PR, it can be forward

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**